PyViCare/PyViCareFloorHeatingDevice.py

```diff
@@ -0,0 +1,17 @@
+from PyViCare.PyViCareHeatingDevice import HeatingDevice
+from PyViCare.PyViCareUtils import handleNotSupported
+
+
+class RadiatorActuator(HeatingDevice):
+
+    @handleNotSupported
+    def getSerial(self):
+        return self.service.getProperty("device.name")["deviceId"]
```

Comment:
Get via `device.name` as long as [no device.serial is provided](https://community.viessmann.de/t5/Feedback-API/Serial-Number-an-Gateways/td-p/410253).